hazelcast/src/main/java/com/hazelcast/map/impl/mapstore/writebehind/CyclicWriteBehindQueue.java

```diff
@@ -180,13 +181,13 @@ public void getFrontByNumber(int numberOfElements, Collection<DelayedEntry> coll
 
     private void addCountIndex(DelayedEntry entry) {
         Data key = (Data) entry.getKey();
-        Map<Data, Integer> index = this.index;
+        Map<Data, MutableInteger> index = this.index;
 
-        Integer count = index.get(key);
+        MutableInteger count = index.get(key);
         if (count == null) {
-            count = 0;
+            count = new MutableInteger();
         }
-        count++;
+        count.value++;
```

Comment:
@pveentjer thought it but this seems very easy to use.
@jerrinot thanks. it will be better to change it in another PR. Only wanted to add `MutableInteger` in this PR.